pkgs/development/interpreters/php/default.nix

```diff
@@ -268,4 +273,11 @@ in {
     # https://bugs.php.net/bug.php?id=76826
     extraPatches = optional stdenv.isDarwin ./php73-darwin-isfinite.patch;
   };
+
+  php74 = generic {
+    version = "7.4.0RC5";
+    sha256 = "1qcnm12b7wgs4i9i0gqa8ff3469wnxjf354jv8kwcakgw2pk57rj";
+
+    extraPatches = [ ./php74-bug-78788.patch ];
```

Comment:
Could you add a comment describing why is this path necessary and link to its source?

Reply:
Added comment.